yt/frontends/ramses/data_structures.py

```diff
@@ -635,6 +659,9 @@ def read_rhs(cast):
 
             self.current_time = (self.time_tot + self.time_simu)/(self.hubble_constant*1e7/3.08e24)/self.parameters['unit_t']
 
+        if self.num_groups > 0:
+            self.group_size = rheader['ncpu'] // self.num_groups
```

Comment:
Will it ever possibly be a problem if, say, the groups do have have identical group sizes?  Is that possible in any situation currently?

Reply:
As of today, it is not possible to have different group sizes, so this should be ok.